### DISCUSSION TITLE
Choosing what apps show in the menu

### DISCUSSION THREAD

**Discussion by JustinGTP** [BimmerGestalt/AAIdrive#529] [Ideas]
Hi! So glad I found this. I have a 2020 mini and thought for sure I had id6 but it's id5 haha. Oh well this gets me Spotify again woohoo.

Just wondering if there's a way to chose what shows. I don't really want to see inshot and Google Play books show in my menu.

Thanks again! And thanks for fixing BMWs problems.

**Comment by hufman** [comment]
Hello fellow Mini driver!
I'm not sure if the protocol accurately tells about ID5/ID6, I've heard other people mention that the information display isn't exactly correct. Mini is special anyways, and I'm not sure how precisely they follow the IDrive versions.

As for the music apps, you can swipe them away in the phone app's Music tab to toggle them, even while connected to the car.

**Reply by JustinGTP** [reply]
Oh thank you so sorry I missed that, I've swiped away the apps I don't need and will try it tomorrow.
